desktop/apps/artwork/lib/auction_label.js

```diff
@@ -0,0 +1,28 @@
+import moment from 'moment'
+import { include } from 'underscore'
+
+function isLiveOpen (status, liveStartAt) {
+  return status === 'open' && moment().isAfter(liveStartAt)
```

Comment:
Can we get this from the `is_live_open` field from metaphysics instead? https://github.com/artsy/metaphysics/blob/1bf01a296a220dffc3f3982b75d2ce2e79a650c9/schema/sale/index.js#L159